src/test/regress/multi_schedule

```diff
@@ -298,6 +298,7 @@ test: foreign_key_to_reference_table
 test: replicate_reference_tables_to_coordinator
 test: coordinator_shouldhaveshards
 test: local_shard_utility_command_execution
+test: coordinator_tables
```

Comment:
> can we run this in parallel with any other test? I guess not because of master_add_node.

yes. 

> So, maybe move this closer to tests which does similar things, and have a single line comment saying that do no try to parallelize this test?

actually most of the tests in those lines (~295-300) shouldn't be parallelized already, do we need to add an additional comment on top of `coordinator_tables`

Reply:
I haven't  seen that, so we're good